elide-standalone/src/test/java/example/ElideStandaloneTest.java

```diff
@@ -298,62 +291,33 @@ public void testAsyncApiEndpoint() throws InterruptedException {
 
                 // Validate GraphQL Response
                 String responseGraphQL = given()
                         .contentType(MediaType.APPLICATION_JSON)
                         .accept(MediaType.APPLICATION_JSON)
                         .body("{\"query\":\"{ asyncQuery(ids: [\\\"ba31ca4e-ed8f-4be0-a0f3-12088fa9263d\\\"]) "
                                 + "{ edges { node { id queryType status result "
-                                + "{ edges { node { id responseBody status} } } } } } }\","
+                                + "{ responseBody httpStatus resultType contentLength } } } } }\","
                                 + "\"variables\":null}")
                         .post("/graphql/api/v1/")
                         .asString();
 
-                String expectedResponse = document(
-                        selections(
-                                field(
-                                        "asyncQuery",
-                                        selections(
-                                                field("id", "ba31ca4e-ed8f-4be0-a0f3-12088fa9263d"),
-                                                field("queryType", "JSONAPI_V1_0"),
-                                                field("status", "COMPLETE"),
-                                                field("result",
-                                                        selections(
-                                                                field("id", "ba31ca4e-ed8f-4be0-a0f3-12088fa9263d"),
-                                                                field("responseBody", "{\\\"data\\\":"
-                                                                        + "[{\\\"type\\\":\\\"post\\\",\\\"id\\\":\\\"2\\\","
-                                                                        + "\\\"attributes\\\":{\\\"abusiveContent\\\":false,"
-                                                                        + "\\\"content\\\":\\\"This is my first post. woot.\\\""
-                                                                        + ",\\\"date\\\":\\\"2019-01-01T00:00Z\\\"}}]}"),
-                                                                field("status", 200)
-                                                        ))
-                                        )
-                                )
-                        )
-                ).toResponse();
-
+                String expectedResponse = "{\"data\":{\"asyncQuery\":{\"edges\":[{\"node\":{\"id\":\"ba31ca4e-ed8f-4be0-a0f3-12088fa9263d\",\"queryType\":\"JSONAPI_V1_0\",\"status\":\"COMPLETE\",\"result\":{\"responseBody\":\"{\\\"data\\\":[{\\\"type\\\":\\\"post\\\",\\\"id\\\":\\\"2\\\",\\\"attributes\\\":{\\\"abusiveContent\\\":false,\\\"content\\\":\\\"This is my first post. woot.\\\",\\\"date\\\":\\\"2019-01-01T00:00Z\\\"}}]}\",\"httpStatus\":200,\"resultType\":\"EMBEDDED\",\"contentLength\":141}}}]}}}";
```

Comment:
Is there a reason we didn't use the DSL?

Reply:
The com.yahoo.elide.contrib.testhelpers.graphql.elements.SelectionSet and com.yahoo.elide.contrib.testhelpers.graphql.elements.Selection  provide the result in edges and nodes. But the embedded query result does not have edges and nodes in the graphql result. I tried to provide the string output from result as the field value, but additional quotes get added during result generation in DSL and I could not get the assertion to work.